tests/api/utils.py

```diff
@@ -120,6 +120,14 @@ def test_cleanup_duplicate(self):
         except ValueError as e:
             self.fail(str(e))
 
+    def test_cleanup_folded_bound_el(self):
+
+        # Open mesh that has folded boundary elements
+        p = Path(__file__).parents[1] / "data" / "before_cleanup_folded_bound_el.2dm"
+        folded_bound_el_mesh = Mesh.open(p, crs=4326)
+        
+        utils.cleanup_folded_bound_el(folded_bound_el_mesh)
+
```

Comment:
@felicio93 it'd be a good idea to add assertions to the test. e.g. to check the correct elements are removed, the final number of elements are expected, the return values are correct, etc.